src/powerbox/tools.py

```diff
@@ -175,9 +226,167 @@ def _get_binweights(coords, weights, bins, average=True, bin_ave=True, log_bins=
     return indx, bins, sumweights
 
 
+def _spherical2cartesian(r, phi_n):
```

Comment:
Can you put a bit more detail in the docstring here... what is the dimensionality of r and phi_n? What angular conventions are used for phi_n?